README.md

```diff
@@ -58,3 +58,77 @@ If you end up wanting to modify an auth context later on, you can do so with `au
 
 ## LiquidWeb Cloud
 The Cloud features you can use in manage.liquidweb.com on your Cloud Servers you can do with this command line tool. See `help cloud` for a full list of features and capabilities.
```

Comment:
this reminds me, we should probably just remove the `CLI interface for LiquidWeb.` line at the top of this readme, that is just a paste of `lw` output with no paramaters so we don't have to keep it up to date everytime we add a new base level command. No real value in having that in the README anyway

Reply:
I don't think that's terrible, but do any other such tools do that?  maybe we should look to them for ideas on how to document ours